### ISSUE TITLE
Provide in-game user guide for technic

### ISSUE THREAD

**Issue by Fixer-007** [minetest-mods/technic#409]
Technic is quite complex, it should have in-game interactive user guide book (with TOC, links, images, navigatable), I believe there are solutions available already.

**Comment by tanmayameher** [comment]
Yes, that would be cool. If the manual will be converted to a book i don't know how many people gonna read that. Because, let's be honest; it's quite lengthy. 📦 

Although, a dictionary kind of book would be awesome to search specific things in that. But still creativity will be your own. It can help with only information.

I have referenced most of the time to players in game or in IRC to this manual. Most of them take a peek and run away telling "it's lengthy" or "i am not good at reading" or "later i will read" etc. xD Some of them do read it though and say a nice thanks to me :) 

So, i think it's basically boils down to something like "patient player who enjoys the journey" vs. "impatient player to gulp it all in one take and get to destination" :P

**Comment by Fixer-007** [comment]
It can be made shorter, book will consists of chapters, each will have "how to craft" picture, short description, and how to use it.

**Comment by twoelk** [comment]
an ingame manuall with pictures would be awesome, or rather a pictureguide with an explaination or two. 
might add some load to media though. 
If this is done a general enough mod I can imagine people would find many uses for such an ingame information source.

**Comment by louisroyer** [comment]
Maybe at least an integration with [`doc` mod](https://repo.or.cz/minetest_doc.git) could provide some basic information about each node uses and some chapters related to `technic` mechanics (how to start, etc.). Sadly, it has no support for pictures yet.
